pkg/openstack/credentials.go

```diff
@@ -119,3 +119,7 @@ func getRequired(secret *corev1.Secret, key string) (string, error) {
 	}
 	return string(value), nil
 }
+
```

Comment:
Do we need this still be as an exported function? I see it is only used in `pkg/apis/openstack/validation/shoot.go`. Can we move it there or remove the function?
If you want to keep it there then please add a doc string.

Reply:
I inlined the function since it's used in one place only and it's a one-liner and amply commented its usage.